### DISCUSSION TITLE
"Date Ranked" only shows loved maps and 4 ranked maps

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22606] [Q&A] [answered]
This worked for me like its supposed to 6 hours ago, then I installed some beatmap packs in stable and reimportet to lazer, now it looks like this. I deleted all beatmaps in lazer before the reimport. This is kind of breaking my mind because not only are there loved maps that have been ranked at some point which I could sort of understand, but also some that are pending, which makes no sense at all.

https://user-images.githubusercontent.com/87947227/218231953-b06d68bd-1219-45b4-b21b-5349e3b62646.mp4

Video only demonstrates part of map collection but the absolute myjority is loved.

### The only ranked ones that show up with that selection applied are always these four, restarting client doesnt change that.

![image](https://user-images.githubusercontent.com/87947227/218232479-f2328dc3-afc5-48a0-8f02-0f66578fa0cc.png)

 ## edit
date submitted has the smae issue i forgot about that.#
edit 2 more downloaded maps downloaded in client from the beatmap listing also show up, but how do I make the game update all the maps I imported?

**Comment by bdach** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/22416
